spec/yaml/swiftnav/sbp/file_io.yaml

```diff
@@ -19,7 +19,7 @@ description: >
 
   Note that some of these messages share the same message type ID for both the
   host request and the device response.
-stable: False
+stable: True
```

Comment:
I don't think we should do this for fileio, the feature was never officially released